Makefile

```diff
@@ -116,6 +116,9 @@ app: ## Bash into App container
 	@echo "Make: Shelling into local application container"
 	@docker-compose -f docker-compose.dev.yml exec server /bin/bash
 
+run-local-db: local-build
+	@docker-compose -f docker-compose.dev.yml up postgres
```

Comment:
What is the reason for adding a command launch just the db?

Reply:
To launch a standalone DB instance without running the whole application. Sometimes standalone DB instance is handy to pre-setup DB for migration.